sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -1,70 +1,5 @@
-
-# Key concepts
-
-Bullet point list of your library's main concepts.
-
-# Examples
-
-Examples of some of the key concepts for your library.
-
-# Troubleshooting
-
-Running into issues? This section should contain details as to what to do there.
-
-# Next steps
-
-More sample code should go here, along with links out to the appropriate example tests.
-
-## Contributing
-For details on contributing to this repository, see the [contributing guide][azure_sdk_for_cpp_contributing].
-
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
-
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-### Additional Helpful Links for Contributors  
-Many people all over the world have helped make this project better.  You'll want to check out:
-
-* [What are some good first issues for new contributors to the repo?](https://github.com/azure/azure-sdk-for-cpp/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22)
-* [How to build and test your change][azure_sdk_for_cpp_contributing_developer_guide]
-* [How you can make a change happen!][azure_sdk_for_cpp_contributing_pull_requests]
-* Frequently Asked Questions (FAQ) and Conceptual Topics in the detailed [Azure SDK for C++ wiki](https://github.com/azure/azure-sdk-for-cpp/wiki).
-
-<!-- ### Community-->
-### Reporting security issues and security bugs
-
-Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) <secure@microsoft.com>. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://www.microsoft.com/msrc/faqs-report-an-issue).
+# Azure Key Vault Keys Package client library for C++
```

Comment:
Add back the boiler plate before submitting.  You can make an issues to track updating the README